pkg/components/util/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package util package has helper functions for installing/deleting/templatising a helm chart. It
```

Comment:
It's a bit odd to have a file just for the docs, but I understand the reasoning. @surajssd is it some pattern you have seen in other projects maybe?

Reply:
In some blogs and projects I have seen that being done. We don't have an extensive documentation here but then I don't see an ideal place to put it.